net.rpdev.OpenTodoList.json

```diff
@@ -0,0 +1,60 @@
+{
+    "id": "net.rpdev.OpenTodoList",
+    "rename-icon": "OpenTodoList",
+    "rename-desktop-file": "OpenTodoList.desktop",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.12",
+    "sdk": "org.kde.Sdk",
+    "command": "OpenTodoList",
+    "finish-args": [
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=wayland",
+        "--filesystem=host",
```

Comment:
Host is a bit too much here, which files a todo list would need that are not on home?

Reply:
Still, you could just give it the right permissions for home, mounted devices & network share instead of the whole stuff.

Reply:
@sat3ll I'm not a flathub maintainer and it's not my decision to accept the app with host permission or not. I was just raising the question here :) But totally agree with what you said :+1:

Reply:
Stupid question (please bear with me, this is my very first attempt on Flatpak): Is there an easy way to override the filesystem access for users? What I have in mind:

* I think it could be okay for 99% of all use cases to reduce the filesystem access to `home`. I still would like to keep it at least at this level, because a valid scenario might be that someone uses e.g. the NextCloud sync client, in which case a user might just create a library in the directory maintained by the NextCloud app.
* If someone needs to access other locations, he could enable this (e.g. if he wants to put a library on a network share).

I am using KDE here and in Discover I see no way to configure such settings. However, if I understand correctly, `flatpak override` might be used for this.

Reply:
You do `flatpak override --filesystem=<host/directory here> net.rpdev.OpenTodoList` as far as I know.

Reply:
I'm just about fine with it, given what the app is trying to do. Would be great if/when we could get some nicer APIs for this.